src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
Don't begin class or method implementation with empty line. Remove all redundant empty lines, be careful :)